vars/govuk.groovy

```diff
@@ -960,7 +963,15 @@ def dockerTagMasterBranch(jobName, branchName, buildNumber) {
 def pushDockerImage(imageName, tagName, asTag = null) {
   tagName = safeDockerTag(tagName)
   docker.withRegistry('https://index.docker.io/v1/', 'govukci-docker-enterprise-hub') {
+    // Push to both accounts until we migrate everything to the new one
     docker.image("govuk/${imageName}:${tagName}").push(asTag ?: tagName)
+    try {
+      docker.image("governmentdigitalservice/${imageName}:${tagName}").ls
```

Comment:
We don't seem to be doing anything with this `.ls` call - if we attempt to push on line 970 and it errors, we catch the error on line 971, so it doesn't look like we're doing any sort of check here.

Probably better to aim for this sort of logic if you can:

```groovy
  if (docker.image("governmentdigitalservice/${imageName}:${tagName}").ls) {
    docker.image("governmentdigitalservice/${imageName}:${tagName}").push(asTag ?: tagName)
  }
  else {
    println("governmentdigitalservice/${imageName} doesn't exist. If you want to push to this org, you must create the repo manually first.")
  }
```

Reply:
I tried doing that but the problem is that it [fails miserably](https://ci.integration.publishing.service.gov.uk/job/smartanswers/job/docker-test/9/console). The ls call is just so the logs look cleaner but not really necessary, so I can remove it.

Reply:
Does it look better now? @ChrisBAshton

Reply:
LGTM 👍 , thanks, Murilo

Could you also please answer the question from https://github.com/alphagov/govuk-jenkinslib/pull/113#pullrequestreview-1132808543? Then I'll be happy to approve. Thanks!

Reply:
I was under the impression that we wanted to create the repos if they don't exist, but I think that has to be done manually anyway so this code should only push to existing repos or show the error message if the repo doesn't exist.

Reply:
We only want to push the 23 repos identified in [the audit](https://trello.com/c/D7kUe72w/2961-investigate-govuk-docker-account-usage). But agreed your code will not push repos if they don't already exist, and the error message looks clear to me 👍